asmcomp/i386/proc.ml

```diff
@@ -107,7 +107,7 @@ let word_addressed = false
    These globals are denoted by Incoming and Outgoing stack locations
    with negative offsets, starting at -64.
    Unlike arguments passed on stack, arguments passed in globals
-   do not prevent tail-call elimination.  The caller stores arguments
+   do not prevent tailcall elimination.  The caller stores arguments
```

Comment:
'tailcall' is not a rarer form. Counting different forms (OCaml-trunk) I've got :  

- ' tail-call ' : 50
- ' tail-calls ' : 23
- ' tailcall ' : 47
- ' tailcalls ' : 27 

So this is fifty-fifty, but 'tailcall' is shorter .  


Out of curiosity, I also checked F# (Visualfsharp-master github). Results are : 

- ' tail-call ' : 20
- ' tail-calls ' : 0
- ' tailcall ' : 451
- ' tailcalls ' : 242

Well, hard to say that 'tailcall' is a rare form (at Microsoft, in any case ...)  

Note : all figures are obtained with :
$ grep -R ' foo ' | wc -w